src/zeit/retresco/convert.py

```diff
@@ -274,6 +274,41 @@ class Gallery(Converter):
     }
 
 
+class Volume(Converter):
+    """This adapter is for indexing actual IVolume objects. Since ICMSContent
+    objects can be adapted to IVolume (finding the associated volume object),
+    we explicitly restrict this here (which is different from the baseclass).
+    """
+
+    interface = zeit.content.volume.interfaces.IVolume
+    grok.name(interface.__name__)
+
+    properties = {
+        'date_digital_published': '',
+        'volume': '',
+        'year': '',
+    }
```

Comment:
Macht Du nen Vorschlag? An `properties` steht bisher dran "Maps CMS name -> TMS name. If no TMS name is given, the CMS name is used." Wie könnte man das klarer machen?